### ISSUE TITLE
Can't summon a friend.

### ISSUE THREAD

**Issue by alastweekdahlman** [dalaranwow/dalaran-wow#5218]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Click summon friend and it then at the end it says "invalid target"
**Expected behaviour**: Tell us what should happen instead
Summon the friend from RaF to your spot.
**Steps to reproduce the problem**:

1. Refer a friend 
2. Friend is in group
3. Click summon friend and it then at the end it says "invalid target"

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

Try by having your friend selected, it will work.

Thank you.

**Comment by alastweekdahlman** [comment]
Tried it and still nothing.

**Comment by robinsch** [comment]
@Lothloryen I fixed the target requirement at some point on old repo.

**Comment by Lothloryen** [comment]
@robinsch Yes but it was not working on every situation. I pushed a quick addition for it long time ago and players usually don't have problems ingame with the summoning so this should not be bugged after both our fixes.

@alastweekdahlman Write an ingame ticket, we will check it out if there is any mistake with your procedure. Also, try without having a target selected, please.
